opensciencegrid/ospool-cm/opt/ospool/update-prios

```diff
@@ -0,0 +1,27 @@
+#!/bin/python3
+
+import re
+import htcondor
+from pprint import pprint
+
+
+def update_user(ad):
+    current_factor = int(ad["PriorityFactor"])
+    new_factor = current_factor
+
+    # ap20/21 migration
+    if re.search("login0[45].osgconnect.net", ad["Name"]):
+        new_factor = 500
+
+    if new_factor != current_factor:
+        user = ad["Name"]
+        print(f"Updating {user} from {current_factor} to {new_factor}")
+        negotiator.setFactor(user, new_factor)
```

Comment:
You should pass the negotiator object as an arg instead of relying on the global scope